src/kbmod/search/RawImage.h

```diff
@@ -10,6 +10,7 @@
 #ifndef RAWIMAGE_H_
 #define RAWIMAGE_H_
 
+#include <algorithm>
```

Comment:
I can't see anything in your changes to RawImage that calls something from this list: https://cplusplus.com/reference/algorithm/

Is this required?

Reply:
The code uses std::sort below, which needs algorithm.

I saw this and the `std::isnan` problems in RawImage.cpp when I tried to build on a completely new platform.